test_01.py

```diff
@@ -33,8 +33,8 @@ def getURL(page):
 		else:
 			tempUrl = baseUrl + str(item)
 			tempList.append(tempUrl)
-		print tempList
-		return tempList
+	print tempList
```

Comment:
Actually this is add  by @anrui811 We can not sure if it's needed by her so in this pr we will keep it ,we can fix it in next ticket i will assign it to @anrui811